### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Fleury14**
@ryekerjh should be pretty close to done, big remaining stuff is on the issues tab along with code cleanup.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -157,6 +156,20 @@ body {
   color: #fff;
 }
 
+/* Hamburger (yum) menu styles */
```
Comment (ryekerjh) [inline css/styles.css:159]: Yum indeed

File: index.html [hunk 2]
```diff
@@ -98,15 +99,22 @@
   </nav>
 
   <div class="container main-container"> <!--Begin main container-->
-    <div class="jumbotron" class="jr-jumbo"> <!-- Begin hero element -->
+    <div class="jumbotron" id="jr-jumbo"> <!-- Begin hero element -->
```
Comment (ryekerjh) [inline index.html:102]: jr-jumbo sounds like a delicious burger

File: index.html [hunk 3]
```diff
@@ -229,21 +251,26 @@ <h3 class="list-group-item-heading">Git / Github</h3>
     <div class="row"> <!-- Begin first thumnail row -->
       <div class="col-sm-6">
         <div class="thumbnail"> <!-- Thumbnail 1 -->
-          <img src="..." alt="...">
+          <div class="embed-responsive embed-responsive-4by3">
```
Comment (ryekerjh) [inline index.html:254]: A little vague with your naming, but it passes

**Review by ryekerjh — COMMENTED** [review]
Overall, very impressive!

File: index.html (hunk 3 above)
Comment (Fleury14) [reply index.html:254]: Should I get in the habit of naming bootstrap classes even if I think I'm only going to use it in one instance? (Ex: add .about-bowling-vid to the embed even if it's the only embed on the page)

**Pull request merged**